### PR TITLE
Add section to README for troubleshooting audio

### PR DESCRIPTION
**Pull request by aaalaniz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,6 +104,60 @@ The following snippet shows an example `build.gradle` with APK splits enabled.
...
+            previousMicrophoneMute = audioManager.isMicrophoneMute();
+            audioManager.setMicrophoneMute(false);
+        } else {
+            audioManager.setMode(previousAudioMode);
+            audioManager.abandonAudioFocus(null);
+            audioManager.setMicrophoneMute(previousMicrophoneMute);
+        }
+    }
+
+### Configuring Hardware Audio Effects
```
Comment (idelgado) [inline README.md]: Why not recommend using the `AudioOptions.Builder()` options when creating the `LocalAudioTrack` instead?

File: README.md [hunk 2]
```diff
@@ -103,6 +104,60 @@ The following snippet shows an example `build.gradle` with APK splits enabled.
 
 The adoption of APK splits requires developers to submit multiple APKs to the Play Store. Refer to [Google’s documentation](https://developer.android.com/google/play/publishing/multiple-apks.html) for how to support this in your application.
 
+## Troubleshooting Audio
+The following sections provide guidance on how to ensure optimal audio quality in your applications.
+
+### Configuring [AudioManager](https://developer.android.com/reference/android/media/AudioManager.html)
```
Comment (idelgado) [inline README.md]: Would prefer moving the link to `AudioManager` to 111 instead of having the link on the header

Comment (aaalaniz) [inline README.md]: Sure

File: README.md (hunk 1 above)
Comment (aaalaniz) [inline README.md]: AudioOptions configures whether you want these audio effects on or off. Not whether they use software or hardware implementations.

Comment (idelgado) [inline README.md]: Aha. 👍

**Pull request merged**